include/midi_sequencer_impl.hpp

```diff
@@ -2250,12 +2250,12 @@ static bool detectRSXX(const char *head, FileAndMemReader &fr)
     bool ret = false;
 
     // Try to identify RSXX format
-    if(head[0] == 0x7D)
+    if (head[0] >= 0x5D && fr.fileSize() > head[0])
```

Comment:
The Code style of this project needs to don't have the space between `if` and the bracket. (1)